e2e/deployers/retry.go

```diff
@@ -45,13 +49,16 @@ func waitSubscriptionPhase(
 func WaitWorkloadHealth(ctx types.TestContext, cluster types.Cluster, namespace string) error {
 	log := ctx.Logger()
 	w := ctx.Workload()
+	startTime := time.Now()
 
 	log.Debugf("Waiting until workload \"%s/%s\" is healthy in cluster %q", namespace, w.GetAppName(), cluster.Name)
 
 	for {
 		err := w.Health(ctx, cluster, namespace)
 		if err == nil {
-			log.Debugf("Workload \"%s/%s\" is healthy in cluster %q", namespace, w.GetAppName(), cluster.Name)
+			elapsed := time.Since(startTime)
```

Comment:
Not sure we need this, we can use `time.Since(start).Seconds()` in the log call, but it looks nice and clear which is the most important thing.